cfme/configure/settings.py

```diff
@@ -349,11 +349,23 @@ def __init__(self, appliance=None):
         Navigatable.__init__(self, appliance=appliance)
 
     @classmethod
-    def set_default_view(cls, button_group_name, default):
-        bg = ButtonGroup(button_group_name)
-        navigate_to(cls, 'All')
-        if bg.active != default:
-            bg.choose(default)
+    def set_default_view(cls, button_group_names, defaults):
+
```

Comment:
**REQUIRED:** For something that seems to have a fair amount of restrictions placed on it below, it would be very useful to have a docstring here, with some example usage. I'm only saying this because the default behaviour (ie having one default that gets expanded for multiple ones seems a little unintuitive.

**OPTIONAL:** I would have preferred to go for tuples for the (button_grp_name, default) and then a fallback default, so that just button_grp_name could be passed if the default was perhaps unchanged. However, I'm happy to merge as is. Just need the docs above.